configs/config-symmetric-only.h

```diff
@@ -8,18 +8,17 @@
  *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
  */
 
+#define MBEDTLS_PSA_CRYPTO_CONFIG_FILE "../configs/crypto-config-symmetric-only.h"
```

Comment:
> How do you foresee these configurations after the crypto split? Would the two repositories have their own reference configurations, or would e.g. `mbedtls/config-symmetric-only.h` reference `TF-PSA-Crypto/config-symmetric-only.h`?

We will have reference configurations in TF-PSA-Crypto. If they are suitable for some mbedtls reference configurations then it seems reasonable to use them. But `mbedtls` could also define its own crypto reference configurations.

Reply:
> would prefer to keep things simple and have all the options in a single file.

When the split is effective and the crypto part is built using the TF-PSA-Crypto build system and configuration file you would need to extract the crypto configuration from the single file or consider that it is fine to use a configuration file that contains TLS/x509 configuration in the TF-PSA-Crypto build system. I think I would prefer to avoid that.